modules/dtj/My-List-Util/.gitignore

```diff
@@ -0,0 +1,17 @@
+Makefile
```

Comment:
Possibly interesting point here: This `.gitignore` was created by `module_starter` by providing the arg `--ignores=git`